spec/spec_helper.rb

```diff
@@ -21,4 +21,4 @@
   config.order = :random
 end
 
-WebMock.disable_net_connect!(allow_localhost: true)
+WebMock.disable_net_connect!(allow_localhost: true, allow: ['json-schema.org'])
```

Comment:
all our schemas refer to the v04 schema at json-schema.org. Alternative to whitelisting it would be to download locally into `/public/schemas`, which I am not opposed to but didn't necessarily want to take on the maintenance burden.